src/components/common/back-button/back-button.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { Location } from '@angular/common';
+
+@Component({
+  selector: 'mes-back-button',
+  templateUrl: 'back-button.html',
+  styleUrls: ['./back-button.scss'],
+})
+export class MesBackButtonComponent {
```

Comment:
can you make the naming consistent, i.e. name the file and folders mes-back-button in keeping with the class and selector?